backend/app/views/spree/admin/payments/_list.html.erb

```diff
@@ -1,4 +1,4 @@
-<table class="index" id='payments' data-order-id='<%= @order.number %>'>
+<table class="index table table-sm table-responsive" id='payments' data-order-id='<%= @order.number %>'>
```

Comment:
Could we not change the table style here? Won’t adding just `table-responsive` be enough to fix the issue?

Reply:
I just tested it and yes, we can use only `table-responsive`, I sent the changes https://github.com/solidusio/solidus/pull/2589/files